src/sidebar/util/time.js

```diff
@@ -25,6 +25,7 @@ export function clearFormatters() {
  * @param {Date} now
  */
 function delta(date, now) {
+  // @ts-ignore
   return now - date;
```

Comment:
Subtracting two `Date`s is possible. They are implicitly converted to a timestamp in milliseconds first. It seems that TS doesn't know this though.

Reply:
One theme I'm seeing in these changes is things that TS doesn't know about (that perhaps it should). I'm hoping these won't be too irritating...